node/defineOptions_test.go

```diff
@@ -897,3 +897,27 @@ func TestWithRequestedItemsHandler_OkRequestedItemsHandlerShouldWork(t *testing.
 	assert.True(t, node.requestedItemsHandler == requestedItemsHeanlder)
 	assert.Nil(t, err)
 }
+
+func TestWithChainID_InvalidShouldErr(t *testing.T) {
+	t.Parallel()
+
+	node, _ := NewNode()
+	opt := WithChainID(nil)
+
+	err := opt(node)
+
```

Comment:
remove empty line

Reply:
done, also renamed the files (requirement from other PRs)